### DISCUSSION TITLE
Limit words count

### DISCUSSION THREAD

**Discussion by GodsonAddy** [ueberdosis/tiptap#4745] [General Questions]
To limit the number of characters in the editor, you use the CharacterCount extension.

```
CharacterCount.configure({
  limit: 240,
})
```

But what if you would like to limit the number of words rather than characters ? Any help will be appreciated. Thanks

**Comment by Dawntraoz** [comment]
It's been a long time since this question was posted, but as it came up in my search results, I wanted to share the solution I found for my own scenario.

I didn't need a character count, only a word count. So, I added word count logic to the text counter, which would now limit based on what you are counting in your text. However, bear in mind that you won't have a character count anymore.

```
CharacterCount.configure({
    limit: validations?.["word-count"]?.max,
    textCounter: (text) => text.split(/\s+/).filter((word) => word !== '').length,
}),
```
